diff/patch file created on Sat, Mar 16, 2019  3:58:16 PM with:
difftar-folder.sh tarballs/Gappa_Tool-gappa-1.3.4.tar.gz Gappa_Tool-gappa-1.3.4 1
TARFILE= tarballs/Gappa_Tool-gappa-1.3.4.tar.gz
FOLDER= Gappa_Tool-gappa-1.3.4
TARSTRIP= 1
TARPREFIX= gappa-1.3.4/
ORIGFOLDER= Gappa_Tool-gappa-1.3.4.orig
--- Gappa_Tool-gappa-1.3.4.orig/src/proofs/property.hpp	2019-03-01 10:08:10.000000000 +0100
+++ Gappa_Tool-gappa-1.3.4/src/proofs/property.hpp	2019-03-16 15:50:26.467975800 +0100
@@ -54,7 +54,7 @@
 class property {
   union store_t {
     char _bnd[sizeof(interval)];
-    long _int;
+    intptr_t _int;
   };
   store_t store;
   interval       &_bnd()       { return *reinterpret_cast< interval       * >(&store); }
@@ -65,16 +65,16 @@
   { assert(real.pred_bnd()); return _bnd(); }
   interval const &bnd() const
   { assert(real.pred_bnd()); return _bnd(); }
-  long &cst()
+  intptr_t &cst()
   { assert(real.pred_cst()); return store._int; }
-  long const &cst() const
+  intptr_t const &cst() const
   { assert(real.pred_cst()); return store._int; }
   property();
   property(ast_real const *);
   property(ast_real const *, interval const &);
   property(predicated_real const &);
   property(predicated_real const &, interval const &);
-  property(predicated_real const &, long);
+  property(predicated_real const &, intptr_t);
   property(property const &);
   property &operator=(property const &);
   ~property();
--- Gappa_Tool-gappa-1.3.4.orig/src/proofs/property.cpp	2019-03-01 10:08:10.000000000 +0100
+++ Gappa_Tool-gappa-1.3.4/src/proofs/property.cpp	2019-03-16 15:50:23.198718900 +0100
@@ -44,7 +44,7 @@
   new(&store) interval(i);
 }
 
-property::property(predicated_real const &r, long i): real(r)
+property::property(predicated_real const &r, intptr_t i): real(r)
 {
   assert(r.pred_cst());
   store._int = i;
